Banking/src/main/java/com/example/Banking/service/AccountDaoImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.example.Banking.service;
+import com.example.Banking.exception.IdNotFound;
+import com.example.Banking.model.Account;
+import com.example.Banking.repository.AccountRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class AccountDaoImpl implements AccountDao {
+
+    @Autowired
+    private AccountRepository accountRepository;
+
+    //To get all accounts
```

Comment:
You don't have to add a comment if the method name or code is self explanatory or does not contain business logic. `AccountDaoImpl.getAll` is pretty obvious returning all accounts

Reply:
Okay